### PR TITLE
[enhancement](memory) print stacktrace for large allocation

### PR DESCRIPTION
**Pull request by dataroaring**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by dataroaring** [comment]
run buildall

File: be/src/runtime/memory/thread_mem_tracker_mgr.h [hunk 1]
```diff
@@ -173,7 +173,7 @@ inline void ThreadMemTrackerMgr::consume(int64_t size) {
     }
     // Large memory alloc should use allocator.h
     // Direct malloc or new large memory, unable to catch std::bad_alloc, BE may OOM.
-    if (size > 4294967296) { // 4G
+    if (size > 1024l * 1024 * 1024) { // 4G
```
Comment (xinyiZzz) [inline be/src/runtime/memory/thread_mem_tracker_mgr.h:176]: modify comment : )

**Review by xinyiZzz — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**